frontend_app_agenda/src/container/Agenda.jsx

```diff
@@ -37,20 +38,20 @@ class Agenda extends React.Component {
     addAllResourceI18n(i18n, this.state.config.translation, this.state.loggedUser.lang)
     i18n.changeLanguage(this.state.loggedUser.lang)
 
-    document.addEventListener('appCustomEvent', this.customEventReducer)
+    document.addEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
   }
 
   customEventReducer = ({ detail: { type, data } }) => { // action: { type: '', data: {} }
     const { state } = this
 
     switch (type) {
-      case 'agenda_showApp':
+      case CUSTOM_EVENT.SHOW_APP(this.state.config.slug):
```

Comment:
please use `stage.config.slug` on this line
It's only a small code cleaning but it increase consistency between apps